### PR TITLE
LPS-97299 Fix pagination with arrows when creating a new form

### PR DESCRIPTION
**Pull request by carolmariaabb**

### CODE REVIEW

**Comment by carolmariaabb** [comment]
ci:test:sf

**Comment by carolmariaabb** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/components/LayoutProvider/LayoutProvider.es.js [hunk 1]
```diff
@@ -461,20 +459,14 @@ class LayoutProvider extends Component {
...
-
-		handlePaginationNextClicked(
-			{
-				activePage,
-				pages
-			},
-			this.dispatch.bind(this)
-		);
+		const pageIndex = Math.min(activePage + 1, pages.length - 1);
+		handlePaginationItemClicked({pageIndex}, this.dispatch.bind(this));
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/components/LayoutProvider/LayoutProvider.es.js]: Hey @carolmariaabb, have you tested this fix against IE 11? I'm afraid that the use of `{` and `}` in `{pageIndex}` can raise problems with this browser. Can you please check it? Thanks.

Comment (carolmariaabb) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/components/LayoutProvider/LayoutProvider.es.js]: Hi @natocesarrego , I just checked and it works fine in IE 11.
Thanks.

Comment (natocesarrego) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/components/LayoutProvider/LayoutProvider.es.js]: Thanks, @carolmariaabb! 🙂

**Comment by natocesarrego** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by natocesarrego** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/84452. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**